core/src/verification/queue/kind.rs

```diff
@@ -98,9 +98,9 @@ pub mod headers {
     pub struct Headers;
 
     impl Kind for Headers {
-        type Input = Header;
-        type Unverified = Header;
-        type Verified = Header;
+        type Input = SyncHeader;
+        type Unverified = SyncHeader;
+        type Verified = SyncHeader;
```

Comment:
I think we don't need the validator set after the header is verified.
How about keeping `Verified` as `Header`?

Reply:
In the next PR, the importer will use SyncHeader to check validator set hash.

The commit: https://github.com/CodeChain-io/foundry/pull/386/commits/1bf7c58edb5b11f17b939e8e56d8d53e3dc2fbf7
The PR: https://github.com/CodeChain-io/foundry/pull/386

Reply:
okay.